pkgs/tools/networking/http-prompt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pythonPackages, httpie }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "0.2.0";
+  name = "http-prompt";
+
+  src = fetchFromGitHub {
+    rev = "893559a55f82a89899b162fad97df1311eb0ba93";
```

Comment:
`rev` should be `v${version}` here; this function simply expands to `github.com/foo/bar/archive/rev.tar.gz`, where `rev` is an arbitrary git ref (version tag or commit id) and is served as an autogenerated tarball. The reason for using a wrapper here is that you avoid the hash changing whenever github decides to use a different compression level or whatever.